QUARKUS-5615.md

```diff
@@ -0,0 +1,33 @@
+Generate Jackson serializers and deserializers
+## Links to the resources
+- JIRA ticket: https://issues.redhat.com/browse/QUARKUS-5615
+- Serialization:
+    - PR: https://github.com/quarkusio/quarkus/pull/41063
+    - Follow-up to the original PR: https://github.com/quarkusio/quarkus/pull/46211
+    - Documentation PR: https://github.com/quarkusio/quarkus/pull/42289
+- Deserialization:
+    - PR: https://github.com/quarkusio/quarkus/pull/42954
+    - Documentation PR: https://github.com/quarkusio/quarkus/pull/46182
+- Related guide: https://quarkus.io/guides/rest#reflection-free-jackson-serialization
+
+## Scope of the testing 
+This feature adds a new Quarkus property (`quarkus.rest.jackson.optimization.enable-reflection-free-serializers`) which should not affect behavior of Quarkus.
+If this property is activated for a project which uses Jackson for reactive rest endpoints, but doesn't use advanced Jackson annotations (anything, besides @JsonProperty and @JsonIgnore), then Quarkus will use compile-time (de)serialisation, which is a bit faster, than the runtime-one.
+
+## Getting familiar with the feature
+- Read the guides:
+ * https://quarkus.io/guides/rest#reflection-free-jackson-serialization
+ * https://quarkus.io/blog/quarkus-metaprogramming/
+
+
+## Community status of the extension/feature:
+Affected extension ('quarkus-rest-jackson') is supported, but the feature itself is in Technology Preview.
+
+## Automated test development
+To get maximum coverage against breaking changes, add this property to every module of main TS, which uses Jackson. Add test method (one per module), which detects, that serialisation class was generated. Make sure, that this test doesn't affect compilation (avoid using reflection, try checking folder with generated classes instead). Make sure, that these tests run in native mode ()
+
+#### Impacted resources
+The building time may become slightly longer, while current tests should run slightly faster. Both of these changes should be negligible for our use cases. One more test per module may add some time to compilation and running times and consume more RAM, but the impact should be minimal.   
+
```

Comment:
@rsvoboda could you elaborate on Qute interoperability? I thought Qute doesn't use json anywhere.

Reply:
https://quarkus.io/guides/qute-reference#vertx_integration has an example with `JsonObject`